component/main.jsonnet

```
@@ -4,6 +4,14 @@ local inv = kap.inventory();
 
 local params = inv.parameters.resource_locker;
 
+local namespace = kube.Namespace(params.namespace) {
+  metadata+: {
+    labels+: {
+      'openshift.io/user-monitoring': 'false',
```

I'd suggest setting `openshift.io/cluster-monitoring=true` instead of this, as there's an open issue regarding the missing cluster-monitoring label (#12)

```suggestion
      'openshift.io/cluster-monitoring': 'true',
```

This should also ensure that the namespace is excluded from user workload monitoring, while also allowing the service monitor to be picked up by cluster-monitoring until we've decided how we want to move forward with monitoring our infra components on OpenShift 4.

---

We tried this, but this resulted in other issues (